data/datapackage.yaml

```diff
@@ -0,0 +1,338 @@
+# --------
+# metadata: annotations.csv references.csv library.csv insertions.csv
+# --------
+
+profile: data-package
+resources:
+  - encoding: utf-8-sig
+    format: csv
+    hashing: md5
+    name: annotations
+    path: annotations.csv
+    profile: tabular-data-resource
+    description: A collection of all written annotations in Jacques Derrida's library
+    schema:
+      primaryKey: id
+      foreignKeys:
+        - fields: book_id
+          reference: 
+            resource: library
+            fields: id
+      fields:
+        - name: id
+          type: string
+          description: Unique identifier linking to the annotation's page
+          format: uri
+          constraints:
+            required: true
+            unique: true
+        - name: book_id
+          type: string
+          format: uri
+          description: Link to the book where the annotation was found
+        - name: book_title
+          type: string
+          description: Title of the book where the annotation was found
+        - name: book_type
+          type: string
+          description: Whether the work is a full book or a section of a book. 
+          constraints:
+            pattern: "Book|Book Section"
+        - name: page
+          type: string
+          description: The page where the annotation was found # QUESTION: What do brackets mean?
+        - name: tags
+          type: string
+          description: Supplemental information about the annotation (e.g. blue ink, arrow)
+        - name: text_content
+          type: any
+          description: Words included in the annotation
+        - name: text_language
+          type: any
+          description: The language used in the annotation
+        - name: text_language_code
+          type: any
+          description: The ISO 639-1 codes for the language used in the annotation
+        - name: text_translation
+          type: any
+          description: 'Translation of the annotation text'
+        - name: quote_content
+          type: string
+          description: The book's text that was highlighted or underlined by the given annotation
+        - name: quote_language
+          type: string
+          description: The language of the book's text that was highlighted by the annotation
+        - name: quote_language_code
+          type: string
+          description: The ISO 639-1 codes for the book's text that was highlighted by the annotation
+        - name: annotator
+          type: string
+          description: The person who made the annotation # QUESTION: Add language about certainty? How did we know this?
```

Comment:
I don't know the certainty or how the researchers knew this, and IDK if it's documented anywhere we can refer to. There may have been some specific ones they knew were others, or maybe it was just an unnecessarily complicated dat model.  FWIW, field help string indicates this could be blank if the annotator was unknown. Do we have any blanks or non-Derrida authors?

Reply:
The unique values:
```
['Derrida, Jacques', 'Derrida, Marguerite', nan]
```
I'll drop the comment. Let me know if there's anything I should do